SlugPath/SlugPath
# Add Open Graph Preview Image

# Scope

Add opengraph images to make preview links shared on social media platforms more appealing.

closes #286 

# How to Test

- Paste link into social media platform and see image preview
- (e.g. linkedIn, Discord, Twitter, Facebook, etc.)

<img width="741" alt="Screenshot 2024-02-26 at 5 53 45 PM" src="https://github.com/SlugPath/SlugPath/assets/20495250/a507ce60-e860-4bbd-b3cf-cf9322befe42">

![image](https://github.com/SlugPath/SlugPath/assets/20495250/7440953d-3aed-473e-836a-304104b7b222)
